homeassistant/components/sunweg/sensor_types/sensor_entity_description.py

```diff
@@ -19,6 +19,5 @@ class SunWEGSensorEntityDescription(SensorEntityDescription, SunWEGRequiredKeysM
     """Describes SunWEG sensor entity."""
 
     api_variable_unit: str | None = None
-    previous_value_drop_threshold: float | None = None
```

Comment:
What's the reason for dropping this?

Reply:
I'm not using this anymore. At first some sensors could change by a small amount, but now they are fine.